drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/SubHeaderBuilder.java

```diff
@@ -101,37 +132,52 @@ public void build() {
             } else {
                 throw new IllegalArgumentException("TODO REMOTE THIS");
             }
-            targetColumnIndex++;
+            upTargetIndex();
         }
     }
 
-    private void makeAction(final int sourceColumnIndex,
-                            final BaseColumn baseColumn) {
+    public void upTargetIndex() {
```

Comment:
The name of this method it does not suggest me what it is really doing.
What do you think about `incrementTargetIndex` or `incrementTargetColumnIndex`?